Frontend/ui-common/src/Engineering/Helpers/Utils.purs

```diff
@@ -258,3 +261,23 @@ catMaybeStrings arr =
       case x of
         Just a -> acc <> a <> " "
         Nothing -> acc) "" arr
+
+capitalizeFirstChar :: String -> String
+capitalizeFirstChar inputStr =
+  let splitedArray = split (Pattern " ") (inputStr)
+      output = map (\item -> (toUpper (take 1 item)) <> (toLower (drop 1 item))) splitedArray
+    in joinWith " " output
+
+capitalizeOnlyFirstChar :: String -> String 
+capitalizeOnlyFirstChar inputStr =
+  let splitedArray = split (Pattern " ") (inputStr)
+      output = map (\item -> (toUpper (take 1 item)) <>  (drop 1 item)) splitedArray
+    in joinWith " " output
+
+fetchLanguage :: String -> String
+fetchLanguage currLang = case currLang of
+                  "HI_IN" -> "hi"
+                  "KN_IN" -> "kn"
+                  "TA_IN" -> "ta"
+                  _       -> "en"
```

Comment:
we don't need this function, we can directly do matching with strings like `"HI_IN"`, if required to send to backend we can use only for that and not for local comparison

Reply:
we are using this for api call